docs/CHANGELOG.md

```diff
@@ -7,10 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+...
+
+## [1.4.0] - 2022-05-02
+
 ### Added
 
 - JSON parse tasks (v2) now support a custom `separator` parameter to substitute for the default `,`.
-- Added `ETH_USE_FORWARDERS` config option to enable transactions forwarding contracts.
+- Log slow SQL queries
+- Fantom and avalanche block explorer urls
+- Display `requestTimeout` in job UI
+- Keeper upkeep order is shuffled
+
+### Fixed
+
+- `LOG_FILE_MAX_SIZE` handling
+- Improved websocket subscription management (fixes issues with multiple-primary-node failover from 1.3.x)
+- VRFv2 fixes and enhancements
+- UI support for `minContractPaymentLinkJuels`
```

Comment:
The `Changed` block below here should be moved up above to `Unreleased`. I think we want this line up there too:
> Added `ETH_USE_FORWARDERS` config option to enable transactions forwarding contracts.

Reply:
Good catch! Addressed in ca3d4bbe245b7c81a4d8e1d6c6f86b0da5dad074 and 305f9db149474da3498170fb3aab4a01b500aa25